doc/buildah.md

```diff
@@ -0,0 +1,68 @@
+Containers Tekton Tasks
+-----------------------
+
+# `Buildah` Tekton Task
+
+The `buildah` Task is meant to build [OCI][OCI] container images without the requirement of container runtime daemon like Docker daemon using [Buildah][containersBuildah], the Task results contain the image name and the SHA256 image digest.
+
+Please, consider the usage example below:
+
+```yaml
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: task-buildah-run-
+spec:
+  pipelineRef:
+    name: task-buildah
+  params:
+    - name: IMAGE
+      value: your-image-name
+    - name: TLS_VERIFY
+      value: true
+    - name: VERBOSE
+      value: false
```

Comment:
For a shorter example, you can omit the Params with default values, like these two.